distribution/lib/Standard/Base/0.0.0-dev/src/Data/Array.enso

```diff
@@ -331,13 +331,41 @@ type Array
        Arguments:
        - function: A function that takes an element in the array and returns
          some transformation of that element.
+       - on_problems: Specifies how to handle any problems that arise in
+         `function`.
+
+       ? Problem Handling
+
+         The result of Errors thrown when executing the function depend on
+         `on_problems`:
+         - Problem_Behavior.Report_Error: The error is thrown.
+         - Problem_Behavior.Report_Warning: The result is `Nothing`, with the
+           error attached as a warning. Currently unimplemented.
+         - Problem_Behavior.Ignore: The result is `Nothing`, and the error
+           attached as a warning.
```

Comment:
```suggestion
         - Problem_Behavior.Ignore: The result is `Nothing`, and the error is ignored.
```

In the current state there does not seem to be any difference between Report_Warning and Ignore?

Reply:
Done